src/IdentityManager/DataService/Extensions/ConfigurationExtension.cs

```diff
@@ -0,0 +1,33 @@
+﻿using IdentityManager.DataService.Configs;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Travely.IdentityManager.Repository.Model.Context;
+
+namespace IdentityManager.DataService.Extensions
+{
+    public static class ConfigurationExtension
+    {
+        public static void ConfigureSqlContext(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddDbContext<IdentityServerDbContext>(options =>
+             options.UseSqlServer(configuration.GetConnectionString("IdentityServerDB")));
+        }
+
+        public static void InitialConfigIdentityServices(this IServiceCollection services)
```

Comment:
This better to name `AddTravelyIdentityService` and configure `IResourceOwnerPasswordValidator ` and `IProfileService`